diff --git a/Makefile.common b/Makefile.common
index c7d4fc0..ef51379 100644
--- a/Makefile.common
+++ b/Makefile.common
@@ -66,7 +66,7 @@ ifeq (${shell uname}, Darwin)
   # That's for Homebrew libelf and avr-gcc support
   HOMEBREW_PREFIX ?= /usr/local
   ifeq (${shell test -d $(HOMEBREW_PREFIX)/Cellar && echo Exists}, Exists)
-   ifneq (${shell test -d $(HOMEBREW_PREFIX)/Cellar/avr-gcc/ && echo Exists}, Exists)
+   ifneq (${shell test -f $(HOMEBREW_PREFIX)/bin/avr-gcc && echo Exists}, Exists)
     $(error Please install avr-gcc: brew tap osx-cross/homebrew-avr ; brew install avr-libc)
    endif
    ifneq (${shell test -d $(HOMEBREW_PREFIX)/Cellar/libelf/ && echo Exists}, Exists)
@@ -110,7 +110,7 @@ MKDIR		?= mkdir -p
 INSTALL		?= install
 SHELL	 	:= ${shell which bash}
 
-OBJ 		:= obj-${shell $(CC) -dumpmachine}
+OBJ 		:= obj
 LIBDIR		:= ${shell pwd}/${SIMAVR}/${OBJ}
 LDFLAGS 	+= -L${LIBDIR} -lsimavr -lm
 
