diff --git a/examples/parts/ssd1306_virt.h b/examples/parts/ssd1306_virt.h
index fdf8103..f2d565d 100644
--- a/examples/parts/ssd1306_virt.h
+++ b/examples/parts/ssd1306_virt.h
@@ -40,6 +40,10 @@
 #ifndef __SSD1306_VIRT_H__
 #define __SSD1306_VIRT_H__
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 #include "sim_irq.h"
 
 #define SSD1306_VIRT_DATA			1
@@ -201,4 +205,8 @@ ssd1306_connect (ssd1306_t * part, ssd1306_wiring_t * wiring);
 void
 ssd1306_connect_twi (ssd1306_t * part, ssd1306_wiring_t * wiring);
 
+#ifdef __cplusplus
+};
+#endif
+
 #endif
